tests/tests.sh

```diff
@@ -99,6 +99,11 @@ function test_websocket {
                "(You may want to increase test timeout)"; FAILURE=1; }
 }
 
+function selenium_test_websocket {
+    pip install selenium
```

Comment:
`pip install selenium` is a one-off job / management job.
Please move it `.travis.yml` it shouldn't run everytime this function is called! :smile: